main/menus.js

```diff
@@ -75,7 +74,12 @@ const sendFeedbackItem = {
 
 const aboutItem = {
   label: `About ${app.name}`,
-  click: openAboutWindow
+  click: () => {
+    closeAllCroppers();
+    ensureDockIsShowing(() => {
+      showAboutWindow({copyright: 'Copyright © Wulkano'});
```

Comment:
It's better to set the options upfront with https://www.electronjs.org/docs/api/app#appsetaboutpaneloptionsoptions at launch. You also don't need the `electron-util` method. It's only useful for cross-platform support.